Add sensors for AsusWRT using http(s) library (#34511)
home-assistant/home-assistant.github.io: source/_integrations/asuswrt.markdown

- Upload sensor (unit_of_measurement: Gigabyte - *Daily accumulation*)
- Upload Speed sensor (unit_of_measurement: Mbit/s)
- Load average sensors (1min, 5min, 15min)

> - Temperature sensors (2ghz, 5ghz, 6ghz, cpu). NB: only temperature sensors available on your router will be created

Reviewer: ```suggestion
- Temperature sensors (2 GHz, 5 GHz, 6 GHz, CPU). Only temperature sensors available on your router will be created.
```

> When integration is configured to use Http(s) protocol, also this sensors will be available:

Reviewer: ```suggestion
If the integration is configured to use the http(s) protocol, also the following sensors will be available:
```

- CPU usage sensors (percentage for total and single core)
- Memory usage sensor (percentage)

> - Memory free sensor (unit_of_measurement: Megabyte)

Reviewer: ```suggestion
- Free memory sensor (Megabyte)
```

> - Memory used sensor (unit_of_measurement: Megabyte)

Reviewer: ```suggestion
- Memory used sensor (Megabyte)
```

- Last boot sensor (Timestamp)

> - Uptime sensor (Seconds)

Reviewer: seconds as in hours, minutes, and seconds? Or seconds as in it will show 68400 if it's been running for a day?

Author: Not clear to me if this should be what HA interface show or how sensor is defined. Sensor definition say `native_unit_of_measurement=UnitOfTime.SECONDS` 

and this is the information in HA sensor properties:

![image](https://github.com/user-attachments/assets/883d80a5-270f-4f9a-8da3-3264c97b85fa)

But in the HA UI is shown as HH:MM:SS

![image](https://github.com/user-attachments/assets/bdc1dcc0-22e1-475e-ac13-5158d05014c2)

(Just reboot my router about 40 minutes ago)

Reviewer: ah, ok, sorry. I meant what the users sees. so hours, minutes, and seconds (HH:MM.SS).

Author: Ok, changed as requested

> - Uptime sensor

Reviewer: What is the unit? Minutes? hours and minutes?
